pkg/lib/cockpit-components-plot.scss

```diff
@@ -0,0 +1,36 @@
+.ct-plot {
+    // Ensure there's no vertical-align offset (set in PF4, for icons)
+    vertical-align: 0;
+
+    &-title {
+        font-size: var(--pf-global--FontSize--md);
+    }
+
+    // Placeholder string to stretch the column, set offscreen
+    &-widest {
+        fill: transparent;
+    }
+
+    &-unit {
```

Comment:
This is duplication below, no?

Reply:
Yep. I pushed what I had.

For debugging purposes, I had all of them separate, including the x and y classes. It's fixed locally, with a bunch of other changes too. I'll push up my current changes—with a fix for this—soon.